lib/ex_machina.ex

```diff
@@ -13,7 +13,25 @@ defmodule ExMachina do
     defexception [:message]
 
     def exception(factory_name) do
-      message = "No factory defined for #{inspect factory_name}"
+      message =
+        """
+        No factory defined for #{inspect factory_name}. This may be because you
+        defined a factory with attrs like this: def factory(:my_factory, attrs)
```

Comment:
Maybe this could be clarified a bit

"defined a factory with 2 parameters like this:"